packages/react-instantsearch-hooks-web/src/widgets/RefinementList.tsx

```diff
@@ -109,16 +132,22 @@ export function RefinementList({
         onReset={onReset}
         onSubmit={onSubmit}
         translations={{
-          submitTitle: 'Submit the search query.',
-          resetTitle: 'Clear the search query.',
+          submitButtonTitle: mergedTranslations.submitButtonTitle,
+          resetButtonTitle: mergedTranslations.resetButtonTitle,
         }}
       />
     ),
     noResults:
-      searchable && isFromSearch && items.length === 0 && 'No results.',
+      searchable &&
+      isFromSearch &&
+      items.length === 0 &&
+      mergedTranslations.noResults,
```

Comment:
Interesting, we forgot this translation in the RFC.

Do you think `noResults` is fine? If we get closer to the naming scheme we chose, maybe `noResultsText` is more appropriate, wdyt?